src/components/MediaViewer/MediaViewer.tsx

```diff
@@ -351,500 +366,615 @@ const MediaViewer = ({ resource }: { resource: CloudinaryResource }) => {
   }
 
   return (
-    <div className="h-screen bg-black px-0">
-
-      {/** Modal for deletion */}
-
-      <Dialog open={deletion && ['confirm', 'deleting'].includes(deletion?.state)} onOpenChange={handleOnDeletionOpenChange}>
-        <DialogContent data-exclude-close-on-click={true}>
-          <DialogHeader>
-            <DialogTitle className="text-center">Are you sure you want to delete?</DialogTitle>
-          </DialogHeader>
-          <DialogFooter className="justify-center sm:justify-center">
-            <Button variant="destructive" onClick={handleOnDelete} disabled={deletion?.state === 'deleting'}>
-              {deletion?.state === 'deleting' && (
-                <Loader2 className="h-4 w-4 mr-2 animate-spin" />
-              )}
-              {deletion?.state !== 'deleting' && (
-                <Trash2 className="h-4 w-4 mr-2" />
-              )}
-              Delete
-            </Button>
-          </DialogFooter>
-        </DialogContent>
-      </Dialog>
+    <>
```

Comment:
This looks like a tonne of changes, but it's just a `<Dialog />` added in the returned JSX. I had to wrap it in a fragment with the existing `<div />` which is why it looks like a massive change.